doc/api/diagnostics_channel.md

```diff
@@ -428,6 +428,24 @@ Emitted when server receives a request.
 
 Emitted when server sends a response.
 
+`net.client.socket`
+
+* `socket` {net.Socket}
+
+Emitted when new a client socket.
+
+`net.server.socket`
+
+* `socket` {net.Socket}
+
+Emitted when receive a connection.
+
+`udp.socket`
+
+* `socket` {dgram.Socket}
+
+Emitted when call `'createSocket'`.
```

Comment:
```suggestion
Emitted when the `'createSocket'` event is emitted.
```

Reply:
`createSocket` is a function, not a event 🤔.

Reply:
Done !